Update convert-unmanaged-to-managed-disks.md

pranoybej opened the pull request:
Have done two changes and below are the description of the changes:
1.Removed the 'Start-AzureRmVM' command from "Convert VMs in an availability set" section 2 as Conversion will automatically restart the VM.
2. Added the conversion fail error caused by failed extension in 'Troubleshoot' section.

azurecla commented:
Hi __@tlmii__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

pranoybej committed:
Update convert-unmanaged-to-managed-disks.md

Have done two changes and below are the description of the changes:
1.Removed the 'Start-AzureRmVM' command from "Convert VMs in an availability set" section 2 as Conversion will automatically restart the VM.
2. Added the conversion fail error caused by failed extension in 'Troubleshoot' section.

pranoybej committed:
Update convert-unmanaged-to-managed-disks.md
